modules/tfm/tfm/common/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if(TFM_PARTITION_PLATFORM)
+  set(src ${CMAKE_CURRENT_LIST_DIR}/src)
+  set(nrf_dir ${CMAKE_CURRENT_LIST_DIR}/../../../../)
+
+  target_include_directories(platform_s
+    PUBLIC
+    include
+    ${nrf_dir}/include/tfm
+    )
+
+  target_sources(platform_s
+    PRIVATE
+    ${src}/tfm_platform_system.c
+    ${src}/tfm_ioctl_s_api.c
+    )
+
+  target_include_directories(platform_ns
+    PUBLIC
+    include
+    ${nrf_dir}/include/tfm
+    )
+
+  target_sources(platform_ns
+    PRIVATE
+    ${src}/tfm_ioctl_ns_api.c
+    )
```

Comment:
This file doesn't need to be added this way. It can be added like any other Zephyr file, since nothing in the TFM codebase relies on it. This goes for all the NS code in this PR AFAICT.